DTAConfig/OptionPanels/AudioOptionsPanel.cs

```diff
@@ -10,6 +10,20 @@ namespace DTAConfig.OptionPanels
 {
     class AudioOptionsPanel : XNAOptionsPanel
     {
+        private const int VOLUME_MIN = 0;
+        private const int VOLUME_MAX = 10;
+        private const float VOLUME_SCALE_FLOAT = 10.0f;
```

Comment:
It should be okay always using double. We are not programming in GPU so there's no situation requires a float.

Reply:
I just realized that the UI library requires a float instead of a double as the paramater. Never mind then.